stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -18,7 +18,7 @@
 ///
 //===----------------------------------------------------------------------===//
 
-#if (defined(__ELF__) || defined(__ANDROID__)) && !defined(__HAIKU__)
+#if (defined(__ELF__) || defined(__ANDROID__) || defined(__Fuchsia__)) && !defined(__HAIKU__)
```

Comment:
Out of curiosity: Fuchsia doesn't define `__ELF__`?

Reply:
It does :) I didn't look closely and was adding defines wherever I seen tests for __ANDROID__ early on. We can kill it for android and fuchsia.

Reply:
fixed. also cleaned up the ```__ANDROID__``` check and cleaned up #endif comment.